server/src/main/java/org/cloudfoundry/identity/uaa/oauth/TokenRevocationEndpoint.java

```diff
@@ -29,26 +28,26 @@
 import org.springframework.security.oauth2.provider.error.WebResponseExceptionTranslator;
 import org.springframework.security.oauth2.provider.expression.OAuth2ExpressionUtils;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 
 import java.util.List;
 
 import static org.springframework.http.HttpStatus.OK;
-import static org.springframework.web.bind.annotation.RequestMethod.DELETE;
-import static org.springframework.web.bind.annotation.RequestMethod.GET;
 
 @Controller
 public class TokenRevocationEndpoint implements ApplicationEventPublisherAware {
 
-    protected final Logger logger = LoggerFactory.getLogger(getClass());
+    protected final SanitizedLogFactory.SanitizedLog logger = SanitizedLogFactory.getLog(getClass());
```

Comment:
I'm curious what Sonar issue this fixes? Is there whitespace in the log output?

Should every uaa class use this?

Reply:
missing input validation, can be solved with
https://github.com/cloudfoundry/uaa/blame/develop/server/src/main/java/org/cloudfoundry/identity/uaa/logging/LogSanitizerUtil.java#L16

Similar to other things, e.g. IdentityZoneHolder, yes we can do a replacement everywhere . I did it here because it was clear from sonar. Other exchanges might be without effect